pyproject.toml

```diff
@@ -11,7 +11,7 @@ skips = ["*/*test.py", "*/test_*.py", "*tests/*.py"]
 branch = true
 
 [tool.coverage.report]
-fail_under = 95
+fail_under = 92
```

Comment:
Why it is necessary to diminish the coverage percentage? Wouldn't be better to just add more tests?

Reply:
It is not always better to have higher coverage. More tests means that refactoring or any change will be harder

Reply:
From the guide, I think we are still covered:
https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-coverage

But more tests can be added later.